runtime/compiler/optimizer/IdiomRecognition.cpp

```diff
@@ -1034,7 +1034,7 @@ TR_CISCGraph::makePreparedCISCGraphs(TR::Compilation *c)
 
    int32_t num = 0;
    bool genTRxx = c->cg()->getSupportsArrayTranslateTRxx();
-   bool genSIMD = c->cg()->getSupportsVectorRegisters() && !c->getOption(TR_DisableSIMDArrayTranslate);
+   bool genSIMD = c->cg()->getSupportsVectorRegisters() && !c->getOption(TR_DisableSIMDArrayTranslate) && !c->isOutOfProcessCompilation();;
```

Comment:
I favor grouping all this JITServer shortcomings in one place. Hence, we could set the `TR_DisableSIMDArrayTranslate` option to false in `J9::Options::setupJITServerOptions()`. This will work unless there are other parts of the code where `TR_DisableSIMDArrayTranslate` could stay enabled.
Also, a brief comment indicating why this option is disabled for JITServer would be helpful to the reader.

Reply:
Outside of IdiomRecognition.cpp and IdiomTransformation.cpp, the only other place I see this option being checked is in:
```
J9::Compilation::canTransformConverterMethod(TR::RecognizedMethod rm)
```
```
OMR::Z::TreeEvaluator::arraytranslateEvaluator
```
```
OMR::ValuePropagation::generateArrayTranslateNode
```

So I think we should be okay with moving this out into `J9::Options::setupJITServerOptions()`.

Reply:
Fixed here: a2363071256e93fd7372c45cc7725394d547c194